http/bearer/src/main/java/org/wildfly/security/http/bearer/BearerTokenAuthenticationMechanism.java

```diff
@@ -90,7 +90,7 @@ public void evaluateRequest(HttpServerRequest request) throws HttpAuthentication
                     handleCallback(verifyCallback);
 
                     if (verifyCallback.isVerified()) {
-                        AuthorizeCallback authorizeCallback = new AuthorizeCallback(null, null);
+                        AuthorizeCallback authorizeCallback = new AuthorizeCallback("remoteUser", "remoteUser");
```

Comment:
Hello @keshav-redhat , I must have misunderstood you before when you asked about whether you can provide remoteUser value. This specific class handles bearer token mechanism on the server, so we cannot hard code specific values here. These should be configured in the test if the test requires it.